docs/repo-docs/guides/migrating-from-nx.mdx

```diff
@@ -357,9 +290,9 @@ pnpm install turbo --global
 
 </PackageManagerTabs>
 
-### Step 8: Add a `turbo.json`
+### Step 7: Add a `turbo.json`
 
-Create a `turbo.json` at the root to register your tasks and describe their task dependencies.
+Create a `turbo.json` at the root to register your tasks and describe their task dependencies. Note, this does not yet cover the same functionality as was in the original `nx.json` and is an example of getting started quickly with a subset of capabilities.
```

Comment:
Can you say what tasks are missing?

Reply:
Sure yeah, again this is regarding the argument that Nx requires "more configuration" and this felt like it wasn't clearly established as being intentionally not "apples to apples". 

Here in Step 7 you have a tasks configuration that only deals with building and serving the Next.js application, whereas the `nx.json` we start the process with covers building, serving (both from source and from dist), typechecking and linting.